ansible_base/resource_registry/apps.py

```diff
@@ -10,6 +12,12 @@
 logger = logging.getLogger('ansible_base.resource_registry.apps')
 
 
+def migrations_are_complete():
+    executor = MigrationExecutor(connection)
+    plan = executor.migration_plan(executor.loader.graph.leaf_nodes())
+    return not bool(plan)
```

Comment:
@relrod we would probably want to move this into utils? The same problems this is trying to fix likely exist somewhere in RBAC and maybe others.

Reply:
:+1: In #548 I introduce a lib.utils.db, that might be a good place? (if so, you can create that module here, I can deal with the conflict later)

Reply:
The rebase has merged these